webviews/android/README.md

```diff
@@ -1 +1,3 @@
-# link-android-webview-example
\ No newline at end of file
+# Link Android Webview Example
+## **Deprecated**
+**Notice:** This example is now deprecated. Please switch to the [Plaid Link Android SDK](https://github.com/plaid/plaid-link-android). This may continue to work for some integrations but new functionality will not be demonstrated. Feature development and general bug fixes have stopped for this directory of the repository.
```

Comment:
nit: Simplify the text hierarchy and wording

```suggestion
## Deprecation Notice
This example is now deprecated. Please switch to the [Plaid Link Android SDK](https://github.com/plaid/plaid-link-android). This may continue to work for some integrations but new functionality will not be demonstrated. Feature development and general bug fixes have stopped for this directory of the repository.
```